main.go

```diff
@@ -1,79 +1,52 @@
 package main
 
 import (
-	"flag"
 	"log"
 	"os"
-	"os/user"
 	"time"
 
 	"go.uber.org/zap"
 
+	k8s "./kubernetes"
```

Comment:
Please fix this line when merge this pr. I can't specify like below now. Because package doesn't exist yet.
```
k8s "github.com/sukeesh/k8s-job-notify/kubernetes"
```

Reply:
Thank you!!